contracts/mock_contract/src/lib.rs

```diff
@@ -117,6 +124,7 @@ mod test {
         contract.instantiate(&InstantiateMsg {}, None, None)?;
         contract.first_message()?;
         contract.second_message("s".to_string(), &[]).unwrap_err();
+        contract.fourth_message().unwrap();
```

Comment:
Wonder why we only check that execute methods compiles, LOL
```suggestion
        contract.fourth_message().unwrap();
        contract.third_query().unwrap();
```

Reply:
Noted that case with coins also missing here, wanna add?

Reply:
Fixed in https://github.com/AbstractSDK/cw-orchestrator/pull/196/commits/06214ce275c7f72419194d45147c8ac0ec1c86ed